### ISSUE TITLE
Understand the requirements of publishing an accessibility statement

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-design-system#800]
## What

Meet with the accessibility team to understand the requirements of an accessibility statement.

## Why

So we publish the right thing.

## Who needs to know about this

Amy

## Further detail
There may be some strict requirements for the content that is required in a statement which should be considered.

There also is work being done elsewhere in GDS to make a format that can be used by services which we should consider finding out about.

- [Public sector website accessibility statements - what you need to know
](https://gds.blog.gov.uk/2018/11/21/public-sector-website-accessibility-statements-what-you-need-to-know/)
- [How we’re helping public sector websites meet accessibility requirements](https://gds.blog.gov.uk/2018/09/24/how-were-helping-public-sector-websites-meet-accessibility-requirements/)

This would be part of the [Make sure that the Design System meets WCAG 2.1](https://github.com/alphagov/govuk-design-system/issues/677) epic.

## Done when

- [x] we have understood what the accessibility team recommends services do
- [x] we understand the requirements of the statement

**Comment by kellylee-gds** [comment]
@amyhupe  @nickcolley as we may not be writing the statement straight away, it might be worth breaking this down into smaller stories
